pkg/handlers/redirect.go

```diff
@@ -48,18 +54,23 @@ func HandleHeaderRedirect(w http.ResponseWriter, r *http.Request, opts map[strin
 	w.Write([]byte(`User Unauthorized`))
 }
 
-func redirectParameters(w http.ResponseWriter, r *http.Request, opts map[string]interface{}) (authURLPath, sep, redirectParameter, redirectURL string, redirect bool) {
+func redirectParameters(_ http.ResponseWriter, r *http.Request, opts map[string]interface{}) (authURLPath, sep, redirectParameter, redirectURL string, loginHint string, redirect bool) {
```

Comment:
@PedroDBFlores , @lennartschoch , please do the calculation of the `finalURL` here, in `redirectParameters()` function (not in `HandleHeaderRedirect()`.

Reply:
I've done the requested changes on the last two commits 👍